components/centraldashboard/public/components/activity-view.js

```diff
@@ -1,112 +1,109 @@
+                    @apply --layout-fit;
                     --accent-color: #007dfc;
                     --primary-background-color: #003c75;
                     --sidebar-default-color: #ffffff4f;
                     --border-color: #f4f4f6;
+                    background: #f1f3f4;
+                    min-height: 50%;
                     padding: 1em;
                 }
                 paper-progress {
                     width: 100%;
                     --paper-progress-active-color: var(--accent-color)
                 }
-                #list {
-                    overflow-y: scroll;
-                }
-                .activity-row {
+                p {
                     background: #fff;
                     border-top: 1px solid rgba(0,0,0,.12);
                     box-shadow: 0 3px 3px rgba(0,0,0,.12);
                     transition: margin .2s cubic-bezier(0.4, 0, 0.2, 1);
                     font-size: 13px;
+                    margin: 0px;
                     padding: 5px;
                 }
-                .activity-row div {
-                    padding: 0 5px;
-                }
-                .event {
-                    flex-basis: 50%;
-                }
-                .event-icon {
-                    height: 18px;
-                    width: 18px;
-                    padding: 0 10px;
-                }
-                .event-icon.build {
-                    transform: scale(-1, 1);
-                    color: rgb(28, 175, 199);
-                }
-                .event-icon.error {
-                    color: rgb(218, 66, 54);
-                }
                 [hidden] {
                     display: none;
                     opacity: 0;
                     pointer-events: none
                 }
             </style>
-            <iron-ajax auto url="/api/activities" handle-as="json"
-                loading="{{loading}}" on-response="_onResponse">
+            <iron-ajax id="ajax" url="/api/activities/[[namespace]]"
+                handle-as="json" loading="{{loading}}"
+                on-response="_onResponse">
             </iron-ajax>
             <paper-progress indeterminate class="slow"
                 hidden$="[[!loading]]"></paper-progress>
-            <div id="list" role="listbox">
-                <template is="dom-repeat" items="[[activities]]">
-                    <div class="activity-row layout horizontal">
-                        <div class="flex">[[item.formattedTime]]</div>
-                        <div class="flex flex-auto event">
-                            <iron-icon class$="event-icon [[item.icon]]"
-                                icon="[[item.icon]]">
-                            </iron-icon>[[item.event]]
-                        </div>
-                        <div class="flex">[[item.source]]</div>
-                    </div>
-                </template>
-            </div>
+            <p hidden$="[[!message]]">[[message]]</p>
```

Comment:
This should have a class for semantic linkability with the css. IE. Make sense when CSS is being read, to know what this element does.

Good example:
`p.Message-Box` or `p.Error-Message`

Reply:
Understood. This is probably a case of me relying too much on the shadow DOM and private styles not needing as many qualifiers as one would expect to use when using global styles.

Reply:
Changed and added an error handling condition as well. Iron-ajax only fires on-response if the request is successful so I added on-error.